wri/earth-dashboard
# Feature/basic auth

## Overview
Adds basic login flow for the admin pages alongside some syntax fixes. 
- Adds a new service to users for checking auth from token in localStorage
- Update `/sign-in` to correctly handle email/password login POST to control tower and return user data
- On successful sign in, takes user to admin dashboard
- Adds login flow to admin pages: loading -> check auth -> valid show page/failed redirect to sign in

## Testing instructions
- visit the admin page, check to logging in message -> redirected to `/sign-in`
- sign in with email and password -> redirect to admin on success

NOTE: this is a very simple starter and does not handle all navigation options, admin views, and is limited to email and password.
